### ISSUE TITLE
dealer waitlist email exists twice

### ISSUE THREAD

**Issue by binary1230** [magfest/ubersystem#2106]
Just discovered this by accident with some new code,

The dealer waitlist email exists twice in our system:

```
AutomatedEmail(Attendee, 'MAGFest Dealer waitlist has been exhausted', 'dealer_waitlist_exhausted.txt',
               lambda a: 'automatically converted to unpaid discounted badge from a dealer application' in a.admin_notes,
               sender=c.MARKETPLACE_EMAIL)
```

and

```
MarketplaceEmail('{EVENT_NAME} Dealer waitlist has been exhausted', 'dealers/waitlist_closing.txt',
                 lambda g: g.status == c.WAITLISTED,
                 when=after(c.DEALER_WAITLIST_CLOSED))
```

I think the result would be one of them would mask the other one because they index by subject/ident in AutomatedEmails.instances[]

**Comment by binary1230** [comment]
will intro a fix for this as part of #1896

**Comment by kitsuta** [comment]
Did you, uh, not notice that it uses two different templates? And... two
different sets of requirements?

I'm all for seeing if we can't combine them but maybe they're like this for
a reason and just need to be renamed.
